core/unit_test/CMakeLists.txt

```diff
@@ -382,6 +382,24 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;OpenACC;HIP;SYCL)
       list(APPEND ${Tag}_SOURCES2D ${file})
     endforeach()
 
+    set(${Tag}_VIEWSUPPORT)
+    IF (Kokkos_ENABLE_IMPL_MDSPAN)
+      foreach(Name
+        ReferenceCountedAccessor
+        ReferenceCountedDataHandle
```

Comment:
Let's make it clear:
```cmake
set(${Tag}_VIEWSUPPORT ReferenceCountedAccessor ReferenceCountedDataHandle)
list(TRANSFORM ${Tag}_VIEWSUPPORT PREPEND ${dir}/Test${Tag}_View_)
list(TRANSFORM ${Tag}_VIEWSUPPORT APPEND  .cpp)
```

Reply:
I would prefer keeping it the same way we do the other tests here. We can then subsequently change them all to this more concise way of expressing stuff.